bower.json

```diff
@@ -15,10 +15,11 @@
     "output"
   ],
   "dependencies": {
-    "purescript-aff": "^5.0.0",
-    "purescript-foreign": "^5.0.0"
+    "purescript-aff": "main",
+    "purescript-foreign": "master"
   },
   "devDependencies": {
-    "purescript-test-unit": "justinwoo/purescript-test-unit#compiler/0.12"
+    "purescript-assert": "master",
+    "purescript-datetime": "master"
```

Comment:
```suggestion
    "purescript-assert": "^5.0.0",
    "purescript-datetime": "^5.0.0"
```

Reply:
We should also update the repository url to be `https` instead of `git`, on line 9